docs/UserGuide.adoc

```diff
@@ -183,197 +183,282 @@ Example:
 // tag::addperson[]
 ==== Adding a person: `add`
 
-Adds a person to the address book. +
-Command format: `add n/NAME p/PHONE_NUMBER [p/ADDITIONAL_PHONE_NUMBERS]... e/EMAIL a/ADDRESS [f/] [dp/DISPLAY_PHOTO_FILE_PATH] [t/TAG]... [s/SOCIAL_PLATFORM USERNAME]...` +
-Alias: `a`
+To add a person to the address book, use the `add` command.
+
+====
+*Command format*: +
+`[red]#add# [green]#n/NAME p/PHONE_NUMBER [p/ADDITIONAL_PHONE_NUMBERS]... e/EMAIL a/ADDRESS [f/] [dp/DISPLAY_PHOTO_FILE_PATH] [t/TAG]... [s/SOCIAL_PLATFORM USERNAME]...#` +
+
+*Alias*: `a`
+====
 
 ****
 * A person can have more than one phone number (at least one). [Since v1.2] +
 Invalid phone numbers will not be added to the contact,
 and there must be at least one valid phone number entered. +
-* A person can be favorited by including the parameter `f/` +
-* A person can have a display photo imported from the location specified by `DISPLAY_PHOTO_FILE_PATH` [Since v1.5rc]
 * A person can have any number of tags (including zero) +
 * A person can have any number of social media accounts (including zero). [Since v1.2] +
-More information about representing social media accounts can be found in the <<social-media-integrations, Social Media Integrations>> section.
+* More information about representing social media accounts can be found in the <<social-media-integrations, Social Media Integrations>> section.
+* *To mark a person as a favorite contact*:
+** A person can be favorited by including the parameter `f/` +
+* *To add a display photo for a person*:
+** A person can have a display photo imported from the location specified by `DISPLAY_PHOTO_FILE_PATH` [Since v1.5rc]
```

Comment:
Since 1.5 is the actual version (rc is just to denote that this version is a release candidate), it should just be [Since v1.5]

Reply:
Updated ✅